### ISSUE TITLE
How long does it take to train the whole network from the scratch? 

### ISSUE THREAD

**Issue by StringBottle** [open-mmlab/mmdetection#1803]
How long does it take to train the whole network from the scratch? I have implemented a Mask RCNN with MobilnetV3 as a backbone network (+FPN) and tried to trained them on COCO data-set. But the training loss of mask_head and loss_class do not decrease under 0.55 and 0.16 respectively while others are under 0.1. I think the most possible reason is that the whole network has been trained from the scratch. I am kind of lazy... so Is it really necessary to train the backbone network first or is it possible to train the whole network without pretraining? I really appreciate you sharing your experiences on training deep learning models.

**Comment by ZwwWayne** [comment]
Hi @StringBottle ,
We have tried to train mask rcnn from scratch and the schedule should be 9x. You can find the results [here](https://github.com/open-mmlab/mmdetection/blob/master/configs/scratch/README.md), the configs are in the same directory. We recommend you to read the paper cited in the [README.md](https://github.com/open-mmlab/mmdetection/blob/master/configs/scratch/README.md)

**Comment by StringBottle** [comment]
@ZwwWayne Thanks for your reply!! The paper you referred is the one that I was looking for (I forgot the title of the paper). I will train the model with the 9x learning schedule.

**Comment by ZwwWayne** [comment]
It is OK.
This issue is solved thus closed, feel free to reopen it if you have any further questions.

**Comment by ZwwWayne** [comment]
Hi @StringBottle ,
To correct a small mistake, our implementation is 6x schedule rather than 9x as shown in the [README](https://github.com/open-mmlab/mmdetection/blob/master/configs/scratch/README.md). I am sorry if this misleads you before.

**Comment by StringBottle** [comment]
@ZwwWayne 

Thanks for your notice but while training the model for 62 epochs, loss_cls still fluctuate from 0.16 to 0.19. Fortunately, loss_mask is consistently decreasing. I will check the mAP after training for 72 epochs. Thanks.
